app.js

```diff
@@ -10,3 +13,112 @@ const menuLink = document.querySelectorAll('.menu-link');
 menuLink.forEach((e) => e.addEventListener('click', () => {
   body.classList.remove('open-menu');
 }));
+
+//  -----------------POPUP-WINDOW-------------
+
+const works = [
+  {
+    id: 1,
+    title: 'Tonic',
+    image: './images/Snapshoot-Portfolio-1.svg',
+    imageAlt: 'my first work',
+    canopy: ['CANOPY', 'Back End Dev', '2015'],
+    descript: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['html', 'css', 'javaScript'],
+    sourcebtn: '#',
+    liveBtn: '#',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    image: './images/Snapshoo-Portfolio-2.svg',
+    imageAlt: 'my second work project',
+    canopy: ['CANOPY', 'Back End Dev', '2015'],
+    descript: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['html', 'css', 'javaScript'],
+    sourcebtn: '#',
+    liveBtn: '#',
+  },
+  {
+    id: 3,
+    title: 'Tonic',
+    image: './images/Snapshoot-Portfolio-3.svg',
+    imageAlt: 'my third work',
+    canopy: ['CANOPY', 'Back End Dev', '2015'],
+    descript: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['html', 'css', 'javaScript'],
+    sourcebtn: '#',
+    liveBtn: '#',
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories',
+    image: './images/Snapshoot-Portfolio-4.svg',
+    imageAlt: 'my fourth work',
+    canopy: ['CANOPY', 'Back End Dev', '2015'],
+    descript: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['HTML', 'CSS', 'javaScript'],
+    sourcebtn: '#',
+    liveBtn: '#',
+  },
+];
+
+for (let i = 0; i < works.length; i += 1) {
+  const work = works[i];
+
+  portfolio.innerHTML += `
+      <div class="card card-1" id="btn-${work.id}">
+        <img src="${work.image}" alt="${work.imageAlt}">
+        <div class="details">
+            <h2>${work.title}</h2>
+            <div class="canopy">
+                <h4>${work.canopy[0]}</h4>
+                <ul>
+                    <li>${work.canopy[1]}</li>
+                    <li>${work.canopy[2]}</li>
+                </ul>
+            </div>
+            <p class="card-descript">${work.descript}</p>
+            <ul class="languages">
+                <li>${work.languages[0]}</li>
+                <li>${work.languages[1]}</li>
+                <li>${work.languages[2]}</li>
+            </ul>
+            <button data-popup-ref="myPopup">See project</button>
+      </div>
+  `;
+}
+
+const popupsBtn = document.querySelectorAll('[data-popup-ref]');
+
+popupsBtn.forEach((btn) => {
+  btn.addEventListener('click', () => {
+    const popupId = btn.getAttribute('data-popup-ref');
+    const popup = document.querySelector(`[data-popup-id='${popupId}']`);
+
+    // console.log(popup);
+
```

Comment:
- [ ] [OPTIONAL] To keep your code clean I suggest to delete comments and console.logs 😉